packages/apps/auth0-actions/jest.setup.js

```diff
@@ -0,0 +1,16 @@
+// This replicates (some of) the globals that are available in
+// an Auth0 script environment
+
+global.configuration = {
+  API_ROOT: 'test_api_host',
+  CLIENT_KEY: 'test_client_key',
+  CLIENT_SECRET: 'test_client_secret',
+};
+
+class WrongUsernameOrPasswordError extends Error {
```

Comment:
This is duplicated from `databaseScriptGlobals.d.ts`. Is it not possible to share the definition?

Reply:
Unfortunately not really - that one is a type definition, this is a class definition (see that the one in the `.d.ts` file doesn't actually implement the constructor)